stable_baselines3/common/base_class.py

```diff
@@ -228,6 +233,17 @@ def _wrap_env(env: GymEnv, verbose: int = 0, monitor_wrapper: bool = True) -> Ve
     def _setup_model(self) -> None:
         """Create networks, buffer and optimizers."""
 
+    def set_logger(self, logger: Logger) -> None:
+        """Setter for for logger object."""
```

Comment:
Maybe we should document that this will override the settings `tensorboard_log` and (many of the effects of) `verbose` set in `__init__`, what do you think?

Reply:
Very good point. Added a warning